CHANGELOG.md

```diff
@@ -2,8 +2,8 @@
 
 ### 2.4.1 (Next)
 
+* [#125](https://github.com/alexa-js/alexa-app/pull/125): Force new when instantiating alexa.app - [@OpenDog](https://github.com/OpenDog).
 * [#119](https://github.com/alexa-js/alexa-app/pull/119): Moved to the [alexa-js organization](https://github.com/alexa-js) - [@dblock](https://github.com/dblock).
-* Your contribution here.
```

Comment:
More like an encouragement to contribute :) Usually I keep it at the bottom. It's fine, thanks for putting it back.